src/test/resources/retest/recheck/de.retest.web.it.PageFrameIT/page-frame-ChromeDriver.open.recheck/retest.xml

```diff
@@ -185,6 +185,10 @@
 									<key>border-top-width</key>
 									<value xsi:type="xsd:string">2px</value>
 								</entry>
+								<entry>
+									<key>disabled</key>
+									<value xsi:type="xsd:string">false</value>
```

Comment:
This change seems to add a lot (more than 200) entries, where disabled = false. Maybe we can reduce this overhead with more defaults? I assume this would already help:

```yaml
input:
  - disabled: false
```

Reply:
Doesn't work, I've tried.
Since "input" is pretty generic, we have either `input at 'html[1]/body[1]/form[1]/input[1]': disabled: expected="(default or absent)", actual="false"` (https://travis-ci.com/retest/recheck-web/builds/115575377) or `input at 'html[1]/body[1]/form[8]/p[1]/input[2]': disabled: expected="(default or absent)", actual=""` (https://travis-ci.com/retest/recheck-web/builds/115572637). I'd vote for the latter (implemented in this PR).

Reply:
So that whenever it is false, this is actually reflected in the XML.

Reply:
OK, I see. I guess this is because there multiple ways to use the attribute?

```html
<input>

<input disabled>

<input disabled="true">

<input disabled="false">
```

Couldn't the JS ensure that there is always a boolean? Something like:

```javascript
"disabled": node.disabled === "" ? false : node.disabled
```

I believe we could then use the default `false` to not bloat the state/XML?

Reply:
Good idea, I'll try...